docs/CONTRIBUTOR_GUIDE.md

```diff
@@ -47,9 +47,28 @@ To run tests locally, the easiest way to get started is with
 hack/run-tests.sh
 ```
 
+This will run all unit tests and code quality tools.
 To run a single tool independently of the other code checks, have a look at the
 other scripts in [`hack/`](../hack).
 
+In addition, there are integration tests to cover high-level krew functionality.
+To run integration tests, you will need to build the `krew` binary beforehand:
+
+```bash
+hack/make-binaries.sh
+hack/run-integration-tests.sh
+```
+
+The above builds binaries for all supported platforms.
+Building only for your platform produces a slightly different image but will
+work in most circumstances:
+
+```bash
+go build ./cmd/krew
```

Comment:
ehh, I'm a bit wary about `go build` as we might be doing interesting things like static linking that it doesn't  capture.

I would've preferred more if `run-integraton-tests` detected out/bin/krew-$OS_$ARCH, but we can keep it as is for now.

Reply:
Yeah, I wanted to outline an alternative to the current build script but that is probably not necessary.

Using the correct platform binary is absolutely necessary, I'lll add that.

Reply:
I added some logic to determine the correct krew binary for integration tests.